testsuite/tests/typing-warnings/unused_rec.ml

```diff
@@ -0,0 +1,49 @@
+(* TEST
+   * expect
+*)
+
+[@@@ocaml.warning "+39"]
+
+let rec f () = 3;;
```

Comment:
Why is this `Line 3`? Should you add a `;;` after the floating documentation item for clarity?

Reply:
I'm not sure whether `;;` would be enough (perhaps you need an intermediate expect block?).
I'm also not sure what's unclear about it.